lib/network.js

```diff
@@ -2,13 +2,22 @@
 
 const os = require('os');
 
-// Convert IP string to number
-// Signature: [ip]
-//   ip - <string>, (optional), default: '127.0.0.1', IP address
+// Convert IPv4 address from string form to number form
+// Signature: IP_string
+//   IP_string - <string>, (optional), default: '127.0.0.1', IPv4 address in string form
 // Returns: <number>
 const ipToInt = (ip = '127.0.0.1') =>
   ip.split('.').reduce((res, item) => (res << 8) + +item, 0);
 
+// Convert IPv4 address from number form to string form
+// Signature: IP_number
+//   IP_number - <number>, (optional), default: 0, IPv4 address in number form
+// Returns: <string>
+const intToIp = (ipInt = 0) =>
+  [ipInt >>> 24, (ipInt >> 16) & 255, (ipInt >> 8) & 255, ipInt & 255].join(
+    '.'
+  );
```

Comment:
I'd prefer to put 4 parts of address into 4 separate variables and concatenate them without creating array and call join